private/buf/buflsp/symbol.go

```diff
@@ -361,6 +367,41 @@ func getCommentsFromDef(def ast.DeclDef) string {
 	}
 	// Reverse the list and return joined.
 	slices.Reverse(comments)
+	// If the file is a remote dependency, link to BSR docs.
+	if s.file != nil && !s.file.IsLocal() {
+		var hasAnchor, isExtension bool
+		switch def.Classify() {
+		case ast.DefKindMessage, ast.DefKindEnum, ast.DefKindService:
+			hasAnchor = true
+		case ast.DefKindExtend:
+			isExtension = def.AsExtend().Extendee.IsZero()
+		}
+
+		var bsrHost, bsrAnchor, bsrTooltip string
+		if s.file.IsWKT() {
+			bsrHost = "buf.build/protocolbuffers/wellknowntypes"
+		} else if fileInfo, ok := s.file.objectInfo.(bufmodule.FileInfo); ok {
+			bsrHost = fileInfo.Module().FullName().String()
+		}
+		bsrTooltip = string(s.ir.FullName())
+		if !hasAnchor {
```

Comment:
What does `hasAnchor` mean here?

Reply:
Added comments earlier for this, hopefully this is a bit more clear.